Vostok.ClusterClient.Core/IClusterClientConfiguration.cs

```diff
@@ -40,9 +40,15 @@ public interface IClusterClientConfiguration
 
         /// <summary>
         /// <para>An implementation of cluster provider. See <see cref="IClusterProvider.GetCluster"/> for more details.</para>
-        /// <para>This parameter is REQUIRED.</para>
+        /// <para>This parameter or <see cref="AsyncClusterProvider"/> is REQUIRED.</para>
         /// </summary>
         IClusterProvider ClusterProvider { get; set; }
+        
+        /// <summary>
+        /// <para>An implementation of cluster provider. See <see cref="IAsyncClusterProvider.GetClusterAsync"/> for more details.</para>
+        /// <para>This parameter or <see cref="ClusterProvider"/> is REQUIRED.</para>
+        /// </summary>
+        IAsyncClusterProvider AsyncClusterProvider { get; set; }
```

Comment:
We should probably note that AsyncClusterProvider can be null. It's not clear from xmldocs.

Reply:
But `ClusterProvider` either can be null initially and have no annotation :)

Reply:
Yeah, I mixed up the implementation with the interface. LGTM, then.